client/Cargo.toml

```diff
@@ -8,10 +8,16 @@ edition = "2021"
 [dependencies]
 core = { path = "../core" }
 db-manager = { path = "../db-manager" }
+transport = { path = "../transport" }
+db-api = { path = "../db-api" }
+env_logger.workspace = true
+log.workspace = true
 
 anyhow = "1.0.75"
 clap = { version = "4.4.6", features = ["derive"] }
 crossterm = "0.27.0"
 ratatui = { version = "0.23.0", features = ["all-widgets"] }
 rand = "0.8.5"
 shellwords = "1.1.0"
+zmq = "0.10.0"
+
```

Comment:
remove empty line

Reply:
![image](https://github.com/illia-st/it-db-group/assets/75951336/5a160f75-31f5-479d-b8f4-3a9f4ceee636)